data/json/items/comestibles/dairy.json

```diff
@@ -242,6 +242,55 @@
     "fun": 7,
     "vitamins": [ [ "calcium", 16 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "cottage_cheese",
+    "name": { "str_sp": "cottage cheese" },
+    "copy-from": "cheese",
+    "weight": "34 g",
+    "spoils_in": "180 days",
+    "quench": 1,
+    "calories": 391,
+    "container": "plastic_bucket",
+    "description": "Unaged curdled milk and whey, made by draining cheese rather than pressing like cheese curds.  The retained whey keeps the curds loose and soft, giving a somewhat soupy texture to this lower calorie variety of cheese.",
+    "price": 385,
+    "price_postapoc": 150,
+    "vitamins": [ [ "calcium", 32 ], [ "iron", 2 ] ],
+    "fun": 3,
+    "flags": [ "NUTRIENT_OVERRIDE" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "cottage_cheese_with_fruit",
+    "name": { "str_sp": "cottage cheese with fruit" },
+    "copy-from": "cheese",
+    "weight": "34 g",
+    "spoils_in": "180 days",
+    "quench": 1,
+    "calories": 391,
+    "description": "Unaged curdled milk and whey, made by draining cheese rather than pressing like cheese curds. The retained whey keeps the curds loose and soft, giving a somewhat soupy texture to this lower calorie variety of cheese.  By adding fruits to it you've made what might be a full meal.",
```

Comment:
```suggestion
    "description": "Unaged curdled milk and whey, made by draining cheese rather than pressing like cheese curds.  The retained whey keeps the curds loose and soft, giving a somewhat soupy texture to this lower calorie variety of cheese.  By adding fruits to it you've made what might be a full meal.",
```